app/modules/assets/models.py

```diff
@@ -292,3 +297,7 @@ def find(cls, guid):
         if not guid:
             return None
         return cls.query.get(guid)
+
```

Comment:
This was somehow missing. It was called from the rules code but had never been implemented.